packages/contracts/contracts/optimistic-ethereum/OVM/accounts/OVM_ECDSAContractAccount.sol

```diff
@@ -107,7 +134,11 @@ contract OVM_ECDSAContractAccount is iOVM_ECDSAContractAccount {
             );
 
             (address created, bytes memory revertdata) = Lib_ExecutionManagerWrapper.ovmCREATE(
-                transaction.data
+                transaction.data,
+                SafeMath.sub(
```

Comment:
We do this calculation in two places. Can we do it once and assign it to a variable instead?